frontend/viewer/src/home/Server.svelte

```diff
@@ -94,11 +133,22 @@
             {$t`Where are my projects?`}
             <Icon icon="i-mdi-open-in-new" class="size-4" />
           </Button>
+          {#if canDownloadByCode}
+            <br/>
```

Comment:
do we need the br here? can we do the same thing via css/tailwind

Reply:
Don't know off-hand how to do it with CSS, but I can change this so the button goes into its own `<div>` and doesn't line up horizontally.

Done(?) in commit 3af11cb88.